src/Notepads/Strings/pl-PL/Resources.resw

```diff
@@ -586,59 +586,59 @@
     <comment>TextEditor: ContextFlyout "Right-to-Left Reading order" toggle button display text.</comment>
   </data>
   <data name="FontStyle_Italic" xml:space="preserve">
-    <value>Italic</value>
+    <value>Kursywa</value>
     <comment>FontStyle: "Italic"</comment>
   </data>
   <data name="FontStyle_Normal" xml:space="preserve">
-    <value>Normal</value>
+    <value>Normalny</value>
     <comment>FontStyle: "Normal"</comment>
   </data>
   <data name="FontStyle_Oblique" xml:space="preserve">
-    <value>Oblique</value>
+    <value>Pochylony</value>
     <comment>FontStyle: "Oblique"</comment>
   </data>
   <data name="FontWeight_Black" xml:space="preserve">
-    <value>Black</value>
+    <value>Gruba (300%)</value>
     <comment>FontWeight: "Black"</comment>
   </data>
   <data name="FontWeight_Bold" xml:space="preserve">
-    <value>Bold</value>
+    <value>Pogrubiona (200%)</value>
     <comment>FontWeight: "Bold"</comment>
   </data>
   <data name="FontWeight_ExtraBlack" xml:space="preserve">
-    <value>Extra Black</value>
+    <value>Najgrubsza (325%)</value>
     <comment>FontWeight: "ExtraBlack"</comment>
   </data>
   <data name="FontWeight_ExtraBold" xml:space="preserve">
-    <value>Extra Bold</value>
+    <value>Mocno pogrubiona (225%)</value>
     <comment>FontWeight: "ExtraBold"</comment>
   </data>
   <data name="FontWeight_ExtraLight" xml:space="preserve">
-    <value>Extra Light</value>
+    <value>Bardzo cienka (25%)</value>
     <comment>FontWeight: "ExtraLight"</comment>
   </data>
   <data name="FontWeight_Light" xml:space="preserve">
-    <value>Light</value>
+    <value>Cienka (50%)</value>
     <comment>FontWeight: "Light"</comment>
   </data>
   <data name="FontWeight_Medium" xml:space="preserve">
-    <value>Medium</value>
+    <value>Średnia (150%)</value>
     <comment>FontWeight: "Medium"</comment>
   </data>
   <data name="FontWeight_Normal" xml:space="preserve">
-    <value>Normal</value>
+    <value>Normalna (100%)</value>
     <comment>FontWeight: "Normal"</comment>
   </data>
   <data name="FontWeight_SemiBold" xml:space="preserve">
-    <value>Semi Bold</value>
+    <value>Lekko pogrubiona (175%)</value>
     <comment>FontWeight: "SemiBold"</comment>
   </data>
   <data name="FontWeight_SemiLight" xml:space="preserve">
-    <value>Semi Light</value>
+    <value>Lekko cienka (75%)</value>
     <comment>FontWeight: "SemiLight"</comment>
   </data>
   <data name="FontWeight_Thin" xml:space="preserve">
-    <value>Thin</value>
+    <value>Najcieńsza (10%)</value>
```

Comment:
25%

Reply:
//
        // Summary:
        //     Specifies a font weight value of 900.
        //
        // Returns:
        //     A FontWeight with a Weight value of 900.
        public static FontWeight Black { get; }
        //
        // Summary:
        //     Specifies a font weight value of 700.
        //
        // Returns:
        //     A FontWeight with a Weight value of 700.
        public static FontWeight Bold { get; }
        //
        // Summary:
        //     Specifies a font weight value of 950.
        //
        // Returns:
        //     A FontWeight with a Weight value of 950.
        public static FontWeight ExtraBlack { get; }
        //
        // Summary:
        //     Specifies a font weight value of 800.
        //
        // Returns:
        //     A FontWeight with a Weight value of 800.
        public static FontWeight ExtraBold { get; }
        //
        // Summary:
        //     Specifies a font weight value of 200.
        //
        // Returns:
        //     A FontWeight with a Weight value of 200.
        public static FontWeight ExtraLight { get; }
        //
        // Summary:
        //     Specifies a font weight value of 300.
        //
        // Returns:
        //     A FontWeight with a Weight value of 300.
        public static FontWeight Light { get; }
        //
        // Summary:
        //     Specifies a font weight value of 500.
        //
        // Returns:
        //     A FontWeight with a Weight value of 500.
        public static FontWeight Medium { get; }
        //
        // Summary:
        //     Specifies a font weight value of 400.
        //
        // Returns:
        //     A FontWeight with a Weight value of 400.
        public static FontWeight Normal { get; }
        //
        // Summary:
        //     Specifies a font weight value of 600.
        //
        // Returns:
        //     A FontWeight with a Weight value of 600.
        public static FontWeight SemiBold { get; }
        //
        // Summary:
        //     Specifies a font weight value of 350.
        //
        // Returns:
        //     A FontWeight with a Weight value of 350.
        public static FontWeight SemiLight { get; }
        //
        // Summary:
        //     Specifies a font weight value of 100.
        //
        // Returns:
        //     A FontWeight with a Weight value of 100.
        public static FontWeight Thin { get; }

Reply:
It was my creative invention. There is no Polish translation for this

Reply:
> It was my creative invention. There is no Polish translation for this

There is also no Chinese translation for some of them, please just keep the English translation here since people will be able to figure out themselves when downloading the font files from internet. They will likely to have English weight suffix as well.